### ISSUE TITLE
Autocomplete onSelect not fired

### ISSUE THREAD

**Issue by oixan** [primefaces/primeng#11552]
[x ] bug report
[ ] feature request
[ ] support request

**Plunkr Case (Bug Reports)**
https://stackblitz.com/edit/primeng-autocomplete-demo-49evvu?file=src%2Fapp%2Fapp.component.ts

**Current behavior**
Event fired: onBlur

**Expected behavior**
Event fired: onSelect 

**Minimal reproduction of the problem with instructions**
1) Click the autocomplete
2) Digit: i 
3) With mouse click 1 item you prefer

**What is the motivation / use case for changing the behavior?**
The behavior obviously not correct

**Please tell us about your environment:**
Windows 10

* **Angular version:** ~13
* **PrimeNG version:** 13.4.0

* **Browser:** [Chrome]

**Comment by cetincakiroglu** [comment]
Hi @oixan,

We're aware of problems similar to this one and it's an accessibility case. 
Right now we're fully focused on Accessibility. Thanks so much for sharing these with us.
Our main goal is to consider all the components one by one and document the individual accessibility features. In doing so, it is to correct the deficiencies or faulty implementations of that component. For this reason, our team carries out a comprehensive and meticulous study. Of course, it will take some time to complete this process.
Currently, working on the components specified in your attached documentation will indeed be a temporary solution for us and we cannot guarantee that these solutions will not change. Because we examine a lot of accessibility documents and try to make the best decisions for users. At this stage, the best decision that can be made about such issues is to stop the actions to be taken on them for the time being. We will reactivate these issues after we complete our work on accessibility in Prime* component libraries. In this way, we will be able to make better decisions about issues.
Details; https://www.primefaces.org/primeng/accessibility

Thanks a lot for your understanding! 

Best Regards

**Comment by mertsincan** [comment]
Hi,

So sorry for the delayed response! Improvements have been made to many components recently, both in terms of performance and enhancement. Therefore, this improvement may have been developed in another issue ticket without realizing it. You can check this in the documentation. If there is no improvement on this, can you reopen the issue so we can include it in our roadmap? 
Please don't forget to add your feedback as a comment after reopening the issue. These will be taken into account by us and will contribute to the development of this feature. Thanks a lot for your understanding!

Best Regards,

**Comment by ValentinCOC** [comment]
Hi,
I have the same issue in primeng 16.9.1, but I can primary reproduce it in MS Edge. Has the behavior been fixed?
